docs/content/commands/npm-unpublish.md

```diff
@@ -73,6 +86,38 @@ mistakes, unnecessary performance degradation, and malicious input.
 If you don't have a clear idea of what you want to do, it is strongly
 recommended that you do not use this option!
 
+#### `workspace`
+
+* Default:
+* Type: String (can be set multiple times)
+
+Enable running a command in the context of the configured workspaces of the
+current project while filtering by running only the workspaces defined by
+this configuration option.
+
+Valid values for the `workspace` config are either:
+
+* Workspace names
+* Path to a workspace directory
+* Path to a parent workspace directory (will result to selecting all of the
+  nested workspaces)
+
+When set for the `npm init` command, this may be set to the folder of a
```

Comment:
Looks like this was copied over from `npm init` but should be updated to reflect usage for `npm unpublish`

Reply:
No this is automatically generated from the config definition https://github.com/npm/cli/blob/release-next/lib/utils/config/definitions.js#L2079-L2081